etc/kayobe/ansible/prometheus.yml.j2

```diff
@@ -247,18 +247,6 @@ scrape_configs:
 {% endfor %}
 {% endif %}
 
-{% if enable_ironic_prometheus_exporter | bool %}
```

Comment:
Weird - this is in stable/2024.1 but not stable/2023.1

Reply:
There are legitimate issues in this file but they're not relevant to getting CI passing. I've dropped those changes for now and will come back to them